src/main/scala/karazin/scala/users/group/week2/homework/Homework.scala

```diff
@@ -55,8 +61,17 @@ object Homework:
 
     private lazy val g = gcd(abs(x), y)
 
-    override def equals(other: Any): Boolean = ???
+    override def equals(other: Any): Boolean = 

```

Comment:
Every time you implement equals you have to implement hashCode.
https://www.baeldung.com/java-equals-hashcode-contracts
https://alvinalexander.com/scala/how-to-define-equals-hashcode-methods-in-scala-object-equality/